instrumentation/servlet/servlet-5.0/jetty11-testing/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  id("otel.javaagent-testing")
+}
+
+dependencies {
+  testInstrumentation(project(":instrumentation:servlet:servlet-5.0:javaagent"))
+  testInstrumentation(project(":instrumentation:jetty:jetty-8.0:javaagent"))
+  testInstrumentation(project(":instrumentation:jetty:jetty-11.0:javaagent"))
+  testImplementation(project(":instrumentation:servlet:servlet-5.0:testing"))
+
+  // Jetty 11 requires Java 11
+  testLibrary("org.eclipse.jetty:jetty-server:11.0.0")
+  testLibrary("org.eclipse.jetty:jetty-servlet:11.0.0")
+
+  latestDepTestLibrary("org.eclipse.jetty:jetty-server:11.+")
```

Comment:
could you add a comment here ` // see jetty-12-testing module` like we have elsewhere